ts/modules/draggable-points.src.ts

```diff
@@ -0,0 +1,3045 @@
+            point: Point;
+            updateProp?: string;
+        }
+        interface DragDropGuideBoxOptionsObject {
+            className?: string;
+            color?: ColorType;
+            cursor?: string;
+            lineColor?: ColorString;
+            lineWidth?: number;
+            zIndex?: number;
+        }
+        interface DragDropHandleOptionsObject {
+            className?: string;
+            color?: ColorType;
+            cursor?: string;
+            lineColor?: ColorString;
+            lineWidth?: number;
+            pathFormatter?: Function;
+            zIndex?: number;
+        }
+        interface DragDropOptionsObject {
+            draggableX?: boolean;
+            draggableY?: boolean;
+            dragHandle?: DragDropHandleOptionsObject;
+            dragMaxX?: number;
+            dragMaxY?: number;
+            dragMinX?: number;
+            dragMinY?: number;
+            dragPrecisionX?: number;
+            dragPrecisionY?: number;
+            dragSensitivity?: number;
+            groupBy?: string;
+            guideBox?: Dictionary<DragDropGuideBoxOptionsObject>;
+            liveRedraw?: boolean;
+        }
+        interface DragDropPointObject {
+            point: Point;
+            newValues: Dictionary<number>;
+        }
+        interface DragDropPositionObject {
+            chartX: number;
+            chartY: number;
+            guideBox?: BBoxObject;
+            points: Dictionary<SVGAttributes>;
+            prevdX?: number;
+            prevdY?: number;
+        }
+        interface DragHandlesObject {
+            group: SVGElement;
+            point: string;
+        }
+        interface Point {
+            /** @requires modules/draggable-points */
+            getDropValues(
+                origin: DragDropPositionObject,
+                newPos: PointerEventObject,
+                updateProps: Dictionary<Dictionary<Dictionary<string>>>
+            ): Dictionary<number>;
+            /** @requires modules/draggable-points */
+            showDragHandles(): void;
+        }
+        interface PointDragCallbackFunction {
+            (this: Point, event: PointDragEventObject): void;
+        }
+        interface PointDragDropObject {
+            newValues: Dictionary<number>;
+            point: Point;
+        }
+        interface PointDragEventObject {
+            newPoints: Dictionary<PointDragDropObject>;
+            origin: object;
+            preventDefault: Function;
+            target: Point;
+            type: 'drag';
+        }
+        interface PointDragStartCallbackFunction {
+            (this: Point, event: PointDragStartEventObject): void;
+        }
+        interface PointDragStartEventObject extends MouseEvent {
+            updateProp?: string;
+        }
+        interface PointDropCallbackFunction {
+            (this: Point, event: PointDropEventObject): void;
+        }
+        interface PointDropEventObject {
```

Comment:
Both the PointDragEventObject and this PointDropEventObject should probably have an optional `newPoint` and `newPointId` prop, as these are sent in for convenience if there is only one point being updated.